src/main/java/com/zufar/onlinestore/payment/converter/PaymentConverter.java

```diff
@@ -1,11 +1,19 @@
 package com.zufar.onlinestore.payment.converter;
 
-import com.zufar.onlinestore.payment.dto.PaymentDetailsDto;
+import com.zufar.onlinestore.cart.converter.ShoppingSessionItemDtoConverter;
+import com.zufar.onlinestore.cart.entity.ShoppingSessionItem;
+import com.zufar.onlinestore.payment.api.dto.ProcessedPaymentDetailsDto;
 import com.zufar.onlinestore.payment.entity.Payment;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
 
-@Mapper
+import java.util.Set;
+
+@Mapper(uses = ShoppingSessionItemDtoConverter.class , componentModel = MappingConstants.ComponentModel.SPRING)
```

Comment:
maybe better use "spring" in the componentModel instead of MappingConstants.ComponentModel.SPRING ?